components/common/listingCart/ListingCartMini.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <transition v-if="listingCartStore.count" name="slide">
+    <div class="listing-container">
+      <div class="is-inline-flex is-align-items-center">
+        <div class="k-shadow theme-background-color border py-2">
+          <div class="is-inline-flex is-align-items-center mx-5">
+            <div>
+              <b>{{ listingCartStore.count }}</b> items
+            </div>
+            <div class="bar mx-4" />
+            <NeoButton
+              :disabled="!listingCartStore.count"
+              variant="text"
+              no-shadow
+              @click.native="listingCartStore.clear">
+              Clear All
+            </NeoButton>
+            <div class="bar mx-4" />
+            <NeoButton
+              variant="text"
+              no-shadow
+              @click.native="listingCartStore.addAllToCart">
+              Select All
+            </NeoButton>
+          </div>
+        </div>
+        <NeoButton
+          class="height-100"
+          :variant="'k-accent'"
+          @click.native="list">
+          List Items
+        </NeoButton>
+      </div>
+    </div>
+  </transition>
+</template>
+<script setup lang="ts">
+import { NeoButton } from '@kodadot1/brick'
+import { useListingCartStore } from '@/stores/listingCart'
+import { usePreferencesStore } from '@/stores/preferences'
+const listingCartStore = useListingCartStore()
+const preferencesStore = usePreferencesStore()
+
+function list() {
+  preferencesStore.listingCartModalOpen = true
+}
+</script>
+<style scoped lang="scss">
+.listing-container {
+  position: fixed;
+  right: 80px;
+  bottom: 50px;
+  z-index: 998;
+}
+
+.height-100 {
```

Comment:
use `h-full` global css

Reply:
@stephenjason89  why are you resolving conversations that are not yet implemented?

it is up to roileo to resolve this conversation

Reply:
@daiagi There's a problem with h-full
<img width="494" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/39498e6e-be22-415f-b8a9-07cc6abe9695">
<img width="494" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/773e6d94-79cc-400b-80b3-119e301b4e73">


as you can see h-full class is there but not working

<img width="526" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/2611f47e-c028-48c8-b99a-fb45ef5aff2e">

Reply:
its about css specificity, 
1. add `!important` to global `h-full` class
2. delete `height-100` class

Reply:
Indeed, but i didn't want to add !important on classes i didnt create as it might break other layouts relying on the fact that it isn't !important

But if you insist, I'm changing it not to !important

Reply:
Many bulma classes have `!important` and when we create our own it's usually in line with bulma utility classes
When someone puts `h-full` class on something, they expect `height` to be `100%` regardless of any CSS specificity

that's why